### PR TITLE
PR

### PR DESCRIPTION
**Pull request by pointbazaar**

### CODE REVIEW

File: lexer/.gitignore [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+build/*
+build/
+*.o
+
+main
```
Comment (pointbazaar) [inline lexer/.gitignore:6]: remove main and such as they are not present

File: lexer/src/strutils.h [hunk 2]
```diff
@@ -1,8 +1,6 @@
-#include <stdbool.h>
```
Comment (Milo-D) [inline lexer/src/strutils.h:1]: Missing Header Guards.

File: lexer/src/loop.h [hunk 3]
```diff
@@ -1,11 +1,4 @@
 //standard headers
```
Comment (Milo-D) [inline lexer/src/loop.h:1]: Missing Header Guards.

File: lexer/src/main.c [hunk 4]
```diff
@@ -136,23 +150,31 @@ int main(int argc, char** argv) {
     stat(filename,&file_meta);
 
 	if( access(tkn_filename,F_OK) == -1){
```
Comment (Milo-D) [inline lexer/src/main.c:152]: Very inconsistent indentation. Hard to read.

File: lexer/.gitignore (hunk 1 above)
Comment (pointbazaar) [reply lexer/.gitignore]: resolved

File: lexer/src/main.c (hunk 4 above)
Comment (pointbazaar) [reply lexer/src/main.c]: github :(

**Pull request merged**